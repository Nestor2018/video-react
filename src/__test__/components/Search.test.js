import React from 'react';
import { mount } from 'enzyme';
import { create } from 'react-test-renderer';

import Search from '../../components/Search';

describe('<Search/>', () => {
	const search = mount(<Search />);
	test('Render del componente Search', () => {
		expect(search.length).toEqual(1);
	});
});

describe('Search Snapshot', () => {
	const search = create(<Search />);
	test('Comprobar la UI del componente Search', () => {
		expect(search.toJSON()).toMatchSnapshot();
	});
});
