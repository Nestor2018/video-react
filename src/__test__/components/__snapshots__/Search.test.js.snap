// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Snapshot Comprobar la UI del componente Search 1`] = `
<section
  className="main"
>
  <h2
    className="main__title"
  >
    ¿Qué quieres ver hoy?
  </h2>
  <input
    className="input"
    placeholder="Buscar..."
    type="text"
  />
</section>
`;
